examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -57,35 +59,44 @@ export interface IDebugViewProps {
  * For the purposes of this test app, it is useful to be able to see both data sources side-by-side.
  */
 export const DebugView: React.FC<IDebugViewProps> = (props: IDebugViewProps) => {
+	const { model } = props;
+	// Flag that represents the state in which clients are actively fetching external data.
+	const [fetchingExternalData, setFetchingExternalData] = useState(false);
 	return (
 		<div>
 			<h2 style={{ textDecoration: "underline" }}>External Data Server App</h2>
-			<TaskListView model={props.model} />
-			<ExternalDataView />
-			<SyncStatusView />
-			<ControlsView model={props.model} />
+			<ExternalAppForm model={model} setFetchingExternalData={setFetchingExternalData} />
+			<ExternalDataView setFetchingExternalData={setFetchingExternalData} />
+			<SyncStatusView fetchingData={fetchingExternalData} taskList={model.taskList} />
+			<ControlsView model={model} />
 		</div>
 	);
 };
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface IExternalDataViewProps {}
+interface IExternalDataViewProps {
+	setFetchingExternalData: React.Dispatch<React.SetStateAction<boolean>>;
+}
 
 const ExternalDataView: React.FC<IExternalDataViewProps> = (props: IExternalDataViewProps) => {
+	const { setFetchingExternalData } = props;
 	const [externalData, setExternalData] = useState({});
 	useEffect(() => {
 		// Run once immediately to run without waiting.
-		pollForServiceUpdates(externalData, setExternalData).catch(console.error);
+		pollForServiceUpdates(externalData, setExternalData, setFetchingExternalData).catch(
+			console.error,
+		);
 
-		// HACK: Poll every 3 seconds
+		// HACK: Poll every 6 seconds
 		const timer = setInterval(() => {
-			pollForServiceUpdates(externalData, setExternalData).catch(console.error);
-		}, 3000);
+			pollForServiceUpdates(externalData, setExternalData, setFetchingExternalData).catch(
+				console.error,
+			);
+		}, 6000);
 
 		return (): void => {
 			clearInterval(timer);
 		};
-	}, [externalData, setExternalData]);
+	}, [externalData, setExternalData, setFetchingExternalData]);
```

Comment:
Same as the earlier comment with the ExternalDataViewProps -- this is all debug data and we don't want to be tracking the polling and updating for this: https://github.com/microsoft/FluidFramework/pull/13776/files#r1117569262

Reply:
I disagree, this `fetching` sync status is meant to reflect the client currently fetching external data, ie: flicker from idle to fetching every 3 seconds. But I see how it doesn't need to live in the poll function itself.

If it's not meant to be here at all, then I'd argue that there's little value in keeping the `fetching`  sync status. The only other place we would display `fetching` is maybe when the client first initializes and fetches external data, but that'd only flicker for a second on the first load and can be easily missed.
cc: @rohandubal  @ChumpChief

Reply:
This looks like it's reflecting the status of sync'ing the external data with the debug app view, which is not really something we need to highlight (since our customers are not building that part - it's just for debug).

The only fetching/sync'ing that we should highlight is data inbound/outbound from Fluid.

Additional suggestion:  This function signature gets a lot cleaner if it just becomes `fetchLatestExternalData(): TaskData`, and then you don't have to hand out the `externalData` and `setExternalData` -- you can just call them directly here in the useEffect.